tests/test_views.py

```diff
@@ -955,36 +972,6 @@ def test_update_node_with_tags(self):
         self.project.reload()
         assert_equal(self.project.title, 'newtitle')
 
-    # Regression test
```

Comment:
Why was this removed?

Reply:
IIRC it was because the test app redirecting strangely and the test was somewhat pointless because it didn't have cases for an active waffle flag states. I don't know what behavior this checking for because I don't know what the desired behavior was before our waffle flags were on. Also, looking at the logical structure of the test, it's easier just to delete then to adapt the the new behavior, which is tested elsewhere. It mostly check for strings in mako templates we plan to delete anyway.

Reply:
I would lean more towards removing the obviated routes being tested, rather than the entire test, until the entire set of routes is obviated.